src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -612,16 +640,16 @@ public VariableScope[] GetVariableScopes(int stackFrameId)
                 new VariableContainerDetails(this.nextVariableId++, "Scope: " + scope);
             this.variables.Add(scopeVariableContainer);
 
-            var results = await this.powerShellContext.ExecuteCommand<PSVariable>(psCommand, sendErrorToHost: false);
+            var results = await this.powerShellContext.ExecuteCommand<PSObject>(psCommand, sendErrorToHost: false);
```

Comment:
So when querying variables in a remote runspace, I take it we don't get back PSVariable types but PSObject instead?